transport_nantes/mobilito/models.py

```diff
@@ -13,6 +13,19 @@ class MobilitoUser(models.Model):
     """Maintain information about Mobilito users."""
 
     user = models.ForeignKey(User, on_delete=models.PROTECT, related_name="+")
+    # null or user defined
+    _username = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        unique=True,
+        verbose_name="Nom d'utilisateur",
```

Comment:
nom d'utilisateur mobilito

Reply:
Adding "mobilito" could lead the users to question what other usernames they have, wouldn't it ?